src/pybind/mgr/volumes/fs/operations/versions/subvolume_base.py

```diff
@@ -442,6 +442,32 @@ def create_base_dir(self, mode):
         except cephfs.Error as e:
             raise VolumeException(-e.args[0], e.args[1])
 
+    def _get_clone_source_info(self):
```

Comment:
IIRC, didn't we have a similar function somewhere in mgr/volumes?

Reply:
Isn't it similar ? https://github.com/ceph/ceph/blob/main/src/pybind/mgr/volumes/fs/operations/versions/subvolume_v1.py#L660

Reply:
I was searching for `_get_clone_source_info()` and therefore couldn't spot `_get_clone_source`. Some modifications and we can use it. Done.